src/pages/[platform]/build-a-backend/auth/connect-your-frontend/sign-up/index.mdx

```diff
@@ -588,8 +654,21 @@ Your application's users can also sign up using passwordless methods. To learn m
 
 <InlineFilter filters={["angular", "javascript", "nextjs", "react", "react-native", "vue"]}>
 
-{/*  */}
+```typescript
+// Confirm sign up with the OTP received and auto sign in
+const { nextStep: confirmSignUpNextStep } = await confirmSignUp({
+	username: 'james',
+	confirmationCode: '123456',
+});
```

Comment:
Does this example make sense in the WebAuthn Passkeys section? Users cannot sign up with passkeys correct?

Reply:
Yes, only authenticated users can create passkeys. You need a fallback authentication factor to generate a passkey.